solr/solr-ref-guide/src/authentication-and-authorization-plugins.adoc

```diff
@@ -17,17 +17,17 @@
 // specific language governing permissions and limitations
 // under the License.
 
-Solr has security frameworks for supporting authentication and authorization of users. This allows for verifying a user's identity and for restricting access to resources in a Solr cluster.
+Solr has security frameworks for supporting authentication, authorization and auditing of users. This allows for verifying a user's identity and for restricting access to resources in a Solr cluster.
 
-Solr includes some plugins out of the box, and additional plugins can be developed using the authentication and authorization frameworks described below.
+Solr includes some plugins out of the box, and additional plugins can be developed using the authentication, authorization and auditlogging frameworks described below.
```

Comment:
Here's another example of the inconsistency using "audit logging" or "auditlogging".